### PR TITLE
OCT-2125: Metrics E2E

### PR DESCRIPTION
**Pull request by jmikolajczyk**
## Description
Check after [PR-567](https://github.com/golemfoundation/octant/pull/567)
## Definition of Done

1. [ ] If required, the desciption of your change is added to the [QA changelog](https://www.notion.so/octantapp/Changelog-for-the-QA-d96fa3b411cf488bb1d8d9a598d88281) 
2. [ ] Acceptance criteria are met.
3. [ ] PR is manually tested before the merge by developer(s).
    - [ ] Happy path is manually checked.
4. [ ] PR is manually tested by QA when their assistance is required (1).
    - [ ] Octant Areas & Test Cases are checked for impact and updated if required (2).
5. [ ] Unit tests are added unless there is a reason to omit them.
6. [ ] Automated tests are added when required.
7. [ ] The code is merged.
8. [ ] Tech documentation is added / updated, reviewed and approved (including mandatory approval by a code owner, should such exist for changed files).
    - [ ] BE: Swagger documentation is updated.
9. [ ] When required by QA:
    - [ ] Deployed to the relevant environment.
    - [ ] Passed system tests.

---

(1) Developer(s) in coordination with QA decide whether it's required. For small tickets introducing small changes QA assistance is most probably not required.

(2) [Octant Areas & Test Cases](https://docs.google.com/spreadsheets/d/1cRe6dxuKJV3a4ZskAwWEPvrFkQm6rEfyUCYwLTYw_Cc).

### CODE REVIEW

File: client/cypress/e2e/_13metrics.cy.ts [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+          .then($el => $el.css('backgroundColor'))
+          .should('be.colored', '#fafafa');
+        cy.get('[data-test=MetricsEpochHeader__NavigationArrows__rightArrowSvg] path')
+          .then($el => $el.css('fill'))
+          .should('be.colored', '#ebebeb');
+        cy.get('[data-test=MetricsEpochHeader__title]')
+          .invoke('text')
+          .should('eq', isMobile || isTablet ? 'E3 Allocation' : 'Epoch 3 Allocation Window');
+        cy.get('[data-test=MetricsEpochHeader__NavigationArrows__rightArrow]').click({
+          force: true,
```
Comment (aziolek) [inline client/cypress/e2e/_13metrics.cy.ts:99]: Why? `force` means it's not actually clickable for the user, indicating a problem with UI.

File: client/cypress/e2e/_13metrics.cy.ts [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+          .then($el => $el.css('backgroundColor'))
+          .should('be.colored', '#fafafa');
+        cy.get('[data-test=MetricsEpochHeader__NavigationArrows__leftArrowSvg] path')
+          .then($el => $el.css('fill'))
+          .should('be.colored', '#9ea39e');
+        cy.get('[data-test=MetricsEpochHeader__title]')
+          .invoke('text')
+          .should('eq', isMobile || isTablet ? 'E3 Allocation' : 'Epoch 3 Allocation Window');
+        cy.get('[data-test=MetricsEpochHeader__NavigationArrows__leftArrow]').click({
+          force: true,
```
Comment (aziolek) [inline client/cypress/e2e/_13metrics.cy.ts:117]: Why? `force` means it's not actually clickable for the user, indicating a problem with UI.

File: client/cypress/e2e/_13metrics.cy.ts [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+        });
+        cy.wait(1000);
+        cy.get('[data-test=MetricsEpochHeader__NavigationArrows__leftArrow]').click({
+          force: true,
```
Comment (aziolek) [inline client/cypress/e2e/_13metrics.cy.ts:121]: Why? `force` means it's not actually clickable for the user, indicating a problem with UI.

File: client/cypress/e2e/_13metrics.cy.ts [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+          .invoke('text')
+          .should('eq', isMobile || isTablet ? 'E1 Allocation' : 'Epoch 1 Allocation Window');
+        cy.get('[data-test=MetricsEpochHeader__NavigationArrows__leftArrow]')
+          .then($el => $el.css('backgroundColor'))
+          .should('be.colored', '#fafafa');
+        cy.get('[data-test=MetricsEpochHeader__NavigationArrows__leftArrowSvg] path')
+          .then($el => $el.css('fill'))
+          .should('be.colored', '#ebebeb');
+        cy.get('[data-test=MetricsEpochHeader__NavigationArrows__leftArrow]').click({
+          force: true,
```
Comment (aziolek) [inline client/cypress/e2e/_13metrics.cy.ts:133]: Why? `force` means it's not actually clickable for the user, indicating a problem with UI.

Comment (jmikolajczyk) [reply client/cypress/e2e/_13metrics.cy.ts]: Fixed

File: client/cypress/e2e/_13metrics.cy.ts (hunk 3 above)
Comment (jmikolajczyk) [reply client/cypress/e2e/_13metrics.cy.ts]: Fixed

File: client/cypress/e2e/_13metrics.cy.ts (hunk 2 above)
Comment (jmikolajczyk) [reply client/cypress/e2e/_13metrics.cy.ts]: Fixed

File: client/cypress/e2e/_13metrics.cy.ts (hunk 1 above)
Comment (jmikolajczyk) [reply client/cypress/e2e/_13metrics.cy.ts]: Fixed

**Review by aziolek — APPROVED** [review]

**Pull request merged**